pkgs/applications/editors/neovim/default.nix

```
@@ -113,6 +113,8 @@ in
       substituteInPlace src/nvim/CMakeLists.txt --replace "    util" ""
     '';
 
+    NIX_LDFLAGS = optional link-lstdcpp [ "-lstdc++"];
```

Let's just add a comment, as to why this is necessary
```suggestion
    # For treesitter plugins, libstdc++.so.6 will be needed
    NIX_LDFLAGS = [ "-lstdc++"];
```